### PR TITLE
[EDU-3105] feat: authors component shows commit history for file

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

Repurposed ContributorList into AuthorList to show contributing authors pulled from commit history

Goal: test in stage and production envs

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-3105]: https://aziontech.atlassian.net/browse/EDU-3105?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/AuthorList.astro [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+		);
+		margin: var(--avatar-row-margin) var(--avatar-overlap) var(--avatar-row-margin) 0;
+	}
+
+	.avatar-list img,
+	.avatar-list a {
+		display: block;
+		border-radius: 50%;
+		width: var(--avatar-size);
+		height: var(--avatar-size);
```
Comment (lfsigreja) [inline src/components/AuthorList.astro:149]: O custom style do docs trás o  com apenas 70% de max-width (por conta das imagens dentro do mdx), esse componente precisa ser 100% (era para ser assim no anterior, devemos ter esquecido de aplicar lá quando mudamos)

Coloca max-width: 100% e de uma olhada como fica, acredito que seja muito melhor.

**Review by lfsigreja — CHANGES_REQUESTED** [review]

File: src/components/AuthorList.astro [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+		{
+		filteredAuthors.map((item) => (
+			<li>
+			<a href={`https://github.com/${item.login}`}>
+				<img
+				alt={item.login}
+				title={item.login}
+				width="3rem"
+				height="3rem"
+				src={`https://avatars.githubusercontent.com/u/${item.id}?s=64`}
```
Comment (bru-andrade) [inline src/components/AuthorList.astro:100]:
```suggestion
				src={`https://avatars.githubusercontent.com/u/${item.id}?s=32`}
```

File: src/components/AuthorList.astro [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+			var(--avatar-outline-offset) + var(--avatar-outline-width) + var(--avatar-row-spacing) / 2
+		);
+		margin: var(--avatar-row-margin) var(--avatar-overlap) var(--avatar-row-margin) 0;
+	}
+
+	.avatar-list img,
+	.avatar-list a {
+		display: block;
+		border-radius: 50%;
+		width: var(--avatar-size);
```
Comment (bru-andrade) [inline src/components/AuthorList.astro:148]:
```suggestion
		max-width: 100%;
```

**Review by lfsigreja — APPROVED** [review]
LGTM

**Review by gabriel-azion — APPROVED** [review]

**Review by robsongajunior — APPROVED** [review]
LGTM

**Pull request merged**